backend/src/test/conftest.py

```diff
@@ -19,6 +22,13 @@
 # file once more manually, so it would be executed twice.
 STARTUP_DONE = bool(int(os.environ.get("STARTUP_DONE", "0")))
 if not STARTUP_DONE:
+    if SQLService().database_contains_data():
+        # Make sure we don't accidentally delete important data
+        logger.error(
+            f"Database '{conf.postgres.db}' is not empty. The tests will only run given a database without any tables in it."
+        )
+        exit(1)
+
     run_required_migrations()
     startup(reset_data=True)
```

Comment:
Just noticed that this will wipe all project files, which is inconvenient when running the tests during development :(

Reply:
Fixed in the makefile PR: https://github.com/uhh-lt/dwts/pull/308/commits/1628004a079227c492516323e921777827ba8ce9

Reply:
finde ich gut, d.h. aber dass wir Tests jetzt ausschließlich über das makefile starten sollten. wird beim makefile automatisch source .env; source .env.testing ausgeführt?

Reply:
Guter Punkt, `.env.testing` hat noch gefehlt. Hab's jetzt eingefügt.